index.html

```diff
@@ -7,6 +7,7 @@
 <body>
 	<div class="container">
 		<h1>YouTubers' Birthdays</h1>
+		<p>YouTubers on the list: <span class="count">0</span></p>
```

Comment:
So, this will update automatically whenever I add a YouTuber, right?

Reply:
Yes it will

Reply:
Perfect.